src/components/pages/Profile/ProfileCreate.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ProfileForm from './ProfileForm';
+
+const ProfileCreate = () => {
+  return (
+    <div>
+      <ProfileForm />
+    </div>
```

Comment:
Theres no need of div here
```suggestion
      <ProfileForm />
```